tests/sync/test_prior_auth_api.py

```
@@ -0,0 +1,337 @@
+"""Test the prior authorization API functionality"""
+
+import json
+import typing
+import uuid
```

Ruff (0.8.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)